kubernetes/base/config/ejabberd.yaml

```diff
@@ -1,26 +1,29 @@
 language: "en"
 allow_contrib_modules: true
 
+# Secret Macros
+include_config_file: /opt/chat-service/secrets/secrets.yaml
```

Comment:
Definitely confirm that renaming the files works -- last I checked, ejabberd really wanted `.yml`, at least for the base configuration file.